influence_benchmark/config/experiment_config.py

```diff
@@ -183,6 +182,14 @@ def __post_init__(self):
             "lora_dropout",
         ]
         self.accelerate_config.set_gpu_ids(self.devices)
+        if self.effective_batch_size % self.per_device_train_batch_size != 0:
```

Comment:
this seems quite similar to the code in accelerate config, is there a reason we split the code over two places?

Reply:
I had done this first since you wouldn't have access to `per_device_train_batch_size` in the accelerate config, but now that you mention it, it makes more sense to pass it as a param to that function 👍